EIPS/eip-4750.md

```diff
@@ -116,17 +116,19 @@ In addition to container format validation rules above, we extend code section v
     1. Code section is invalid in case offset points to a position outside of section bounds.
     2. Code section is invalid in case offset points to one of two bytes directly following `CALLF` instruction.
 
+### Deprecated instructions
```

Comment:
It uses "deprecated" in abstract, "disallowed" in motivation, "deprecated" here. We should choose one, and "disallowed" sounds closer to what it is.

Reply:
Changed to "disallowed" everywhere